src/bin/tikv-ctl.rs

```diff
@@ -13,20 +13,12 @@
 
 #[macro_use]
 extern crate clap;
-extern crate chrono;
-extern crate futures;
-extern crate grpcio;
-extern crate hex;
-extern crate kvproto;
-extern crate libc;
-#[cfg(unix)]
-extern crate nix;
-extern crate protobuf;
-extern crate raft;
-extern crate rand;
-extern crate rocksdb;
-#[cfg(unix)]
-extern crate signal;
+
+use hex;
```

Comment:
maybe these use can be removed

Reply:
All these are used, we might be able to remove them if we inline the usage in some places, but I didn't try tracking them all down (again this is what Rustfix does)